app/decorators/case/ico/base_decorator.rb

```diff
@@ -2,7 +2,23 @@ class Case::ICO::BaseDecorator < Case::BaseDecorator
 
   attr_accessor :original_case_number, :related_case_number
 
-  def date_decision_received
+  def formatted_date_ico_decision_received
     I18n.l(object.date_ico_decision_received, format: :default)
```

Comment:
why does this method need to be called formatted-date_ico_decision_received, and not just date_ico_decision_recieved?  It is in a decorator, so the whole point is to have the same attribute names.

Reply:
I seem to remember running into difficulties when it came to working out targets because it cant use the formatted version to work out if its in target or not. So thats why it was originally `date_decision_received` but I think this method name is more descriptive and easier to read.

Reply:
I don't quite understand what you're saying, but normally, decorators use the same method name as the original attribute, and just add formatting or prettification.  However, looking at our decorators, we don't seem to have done that anywhere. Go figure!.  Will approve.

Reply:
I know that @aliuk2012 and I were looking at this together. I think @stephenrichards's point is worth considering, even though I advocated for the `formatted_...` version, the Draper docs do seem to steer towards simply using the same method name as what's being decorated and it's true that to get the undecorated version we can just use `object`.